docs-mslearn/focus/conformance.md

```diff
@@ -0,0 +1,854 @@
+| DH3.8   | MUST   | CommitmentDiscountId MUST be set to the ID assigned to the discount \[for commitment discount usage\].                                                                                                                                             | Supports        |                                                                                                                                                                                                                                                                                  |
+| DH3.9   | MUST   | ResourceId MUST be set to the ID of the resource that received the discount \[for commitment discount usage\].                                                                                                                                     | Supports        |                                                                                                                                                                                                                                                                                  |
+| DH3.10  | MUST   | If a commitment is not fully utilized, the provider MUST include a row that represents the unused portion of the commitment for that charge period.                                                                                                | Supports        |                                                                                                                                                                                                                                                                                  |
+| DH3.11  | MUST   | These rows MUST be represented with CommitmentDiscountStatus set to "Unused" and ChargeCategory set to "Usage".                                                                                                                                    | Supports        |                                                                                                                                                                                                                                                                                  |
+| DH3.12  | MUST   | Such rows MUST have their CommitmentDiscountId and ResourceId set to the ID assigned to the commitment-based discount.                                                                                                                             | Not Evaluated   |                                                                                                                                                                                                                                                                                  |
```

Comment:
```suggestion
| DH3.12  | MUST   | Such rows MUST have their CommitmentDiscountId and ResourceId set to the ID assigned to the commitment-based discount.                                                                                                                             | Partially Supports | CommitmentDisocuntId logically matches ResourceId, but they differ in case. |
```